cmd/expired-authz-purger/main.go

```diff
@@ -95,34 +97,55 @@ func (p *expiredAuthzPurger) purge(table string, yes bool, purgeBefore time.Time
 		}
 	}
 
+	wg := new(sync.WaitGroup)
+	work := make(chan string, len(ids))
```

Comment:
nit: this `chan` could replace the `ids` slice up on L49 so that you don't need to have an extra `for` loop below but ¯\_(ツ)_/¯.

Reply:
I don't think I can. Right now I can set the capacity of the chan to `len(ids)`, which is known, to avoid blocking. But at the top of this function I don't yet know how many ids I'll get because it's the result of a series of selects.

Reply:
Doop yup you're right, for some reason I was thinking you can have an unsized non-blocking channel, which you cannot.